test/regress/bibtex

```diff
@@ -0,0 +1,460 @@
+#
+# DO NOT MODIFY!!!!
+# This file is automatically generated by Racc 1.4.13
+# from Racc grammar file "".
+#
+
+require 'racc/parser.rb'
+
+require 'bibtex/lexer'
+
+module BibTeX
+  class Parser < Racc::Parser
+
+module_eval(<<'...end bibtex.y/module_eval...', 'bibtex.y', 89)
+
+  attr_reader :lexer, :options
+
+  @defaults = {
+    :include => [:errors],
+    :allow_missing_keys => false,
+    :debug => false
+  }.freeze
+
+  class << self
+    attr_reader :defaults
+  end
+
+  def initialize(options = {})
+    @options = Parser.defaults.merge(options)
+    @lexer = Lexer.new(@options)
+  end
+
+  def parse(input)
+    @yydebug = debug?
+
+    lexer.analyse(input)
+    do_parse
+    #yyparse(@lexer,:each)
+  end
+
+  def next_token
+    lexer.next_token
+  end
+
+  def debug?
+    options[:debug] || ENV['DEBUG']
+  end
+
+  def allow_missing_keys?
+    options[:allow_missing_keys]
+  end
+
+  def missing_key
+    unless allow_missing_keys?
+      raise ParseError, "Failed to parse BibTeX entry: cite-key missing"
+    end
+  end
+
+  def on_error(tid, val, vstack)
+    message =
+      "Failed to parse BibTeX on value #{val.inspect} (#{token_to_str(tid) || '?'}) #{ vstack.inspect}"
+
+    BibTeX.log.error message
+    raise ParseError, message
+  end
+
+# -*- racc -*-
+...end bibtex.y/module_eval...
+##### State transition tables begin ###
+
+racc_action_table = [
+14, 33, 38, 26, 32, 26, 24, 18, 48, 15, 34, 33, 16, 37, 32, 
```

Comment:
Is there a benefit to not pretty-printing these tables?

The old way makes them look slightly less intimidating